recipes/ddtrace/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ddtrace" %}
+{% set version = "0.42.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ddtrace-{{ version }}.tar.gz
+  sha256: dba66c7013a4352dfa20ed31e989001250056c89811d1cb5ee10e23dc98bac0a
+
+build:
+  number: 0
+  skip: true   # [py==30 or py==31 or py==32 or py==33 or py==34]
+  entry_points:
+    - ddtrace-run = ddtrace.commands.ddtrace_run:main
```

Comment:
It looks like these were specified just for the old tests:

```
  commands:
    - pip check
    - ddtrace-run --help
    - pyddprofile --help
```
Would it be reasonable to remove these?

Reply:
You could probably remove the pip check

Reply:
I removed all of these in a previous diff - these were also generated by grayskull, but it looks like these commands need more complex environment setup to execute.